src/main/resources/application.yml

```diff
@@ -38,6 +38,12 @@ spring.jpa.hibernate.ddl-auto: none
 server.port: ${osiam.port:8080}
 security.oauth2.resource.filterOrder: 3
 
+spring:
+  datasource:
```

Comment:
Configuring the datasource is not needed. As we already define our [own `Datasource` Bean](https://github.com/osiam/osiam/blob/master/src/main/java/org/osiam/configuration/DatabaseConfiguration.java#L60), these values will never be picked up by the respective Auto Configuration.